### DISCUSSION TITLE
Elsa V3 features.

### DISCUSSION THREAD

**Discussion by wfurney** [elsa-workflows/elsa-core#4316] [Q&A]
Hello, 

I am curious how elsa v3 differs from v2. Is this something that's documented somewhere?

**Comment by sfmskywalker** [comment]
Although this list is not exhaustive, I started to add some key new features on this page: https://v3.elsaworkflows.io/

**Reply by wfurney** [reply]
Do you happen to have a link?

**Reply by sfmskywalker** [reply]
Yes, I updated my previous message with the url: https://v3.elsaworkflows.io/

**Reply by wfurney** [reply]
Thanks!
